com.unity.render-pipelines.high-definition/Runtime/Debug/DebugViewMaterialGBuffer.shader

```diff
@@ -119,7 +119,7 @@ Shader "Hidden/HDRP/DebugViewMaterialGBuffer"
                     // TEMP!
                     // For now, the final blit in the backbuffer performs an sRGB write
                     // So in the meantime we apply the inverse transform to linear data to compensate.
-                    if (!needLinearToSRGB)
+                    if (!needLinearToSRGB && _DebugAllowsRGBConversion)
```

Comment:
I guess master isn't merge? @pmavridis have currently implemented similar feature here https://github.com/Unity-Technologies/Graphics/pull/4118/files . so we should rely on _DebugAOVOutput for this. I guess Pavlos simply forget to add _DebugAOVOutput in this file? DebugViewMaterialGBuffer.shader

Reply:
Indeed, I forgot about this file! For AOVs we never hit this code path afaik, that's why it did not cause any issues, but yes, the variable should be here too to allow linear output. I will prepare a PR for master.